app/controllers/peeps.rb

```diff
@@ -0,0 +1,37 @@
+class Chitter < Sinatra::Base
+  get '/peeps' do
+    @peeps = Peep.all.sort_by { |peep| peep.created_at }.reverse
```

Comment:
Style/SymbolProc: Pass `&:created_at` as an argument to `sort_by` instead of a block.